samples/grpc/shopping-analytics-service-scala/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2.2'
+services:
+  postgres-db:
+    image: postgres:latest
+    container_name: postgres_db
```

Comment:
has that been changed in the Java samples also?

Reply:
Yep. But I missed updating the READMEs there